Pokemon Tabletop Adventure/Data Models/WBPokémon.swift

```diff
@@ -34,23 +35,23 @@ struct Pokémon: Codable {
     }
     private enum CodingKeys: String, CodingKey {
         case dexNumber = "id"
-        case name
+        case name = "name"
         case stats = "base_stats"
-        case type
-        case abilities
-        case evolution
-        case biology
+        case type = "type"
```

Comment:
I don't think you need to give specific values unless it doesn't match the case

Reply:
That's what the swiftlint disable at the top was for - there are conflicting rules.
One was for preventing non-assigned enum values, and then that cunt up top for redundant assigned values.